src/types/quizzes/interfaces/quizzes.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { RequestParams, CommonEntityFields } from '~/types'
+
+export interface QuizzesParams extends RequestParams {
+  title: string
+}
+
+export interface Answer {
+  text: string
+  isCorrect: boolean
+}
+
+export interface QuestionWithAnswers {
+  question: string
+  answers: Answer[]
+}
+
+export interface QuizInterface extends CommonEntityFields {
```

Comment:
i think that it would be better if you rename this interface just to `Quiz` as it is with other interfaces of entities

Reply:
Fixed